routes/profile.js

```diff
@@ -0,0 +1,20 @@
+import Router from "express";
+import passport from "passport";
+import Profile from "../controllers/profileController";
+const profileRouter = Router();
+
+profileRouter
+  .put(
+    "/profiles/:username",
+    passport.authenticate("jwt", { session: false }),
+    Profile.update
+  )
+  .get("/profiles/:username", Profile.getProfile)
```

Comment:
We need middleware to check whether the user has been identified for both of your routes @YvesIraguha `isVerified` true or false

Reply:
Those routes, are not protected. There is no middleware needed there.